.NET/Microsoft.Recognizers.Text.Number/NumberRecognizer.cs

```diff
@@ -274,13 +274,11 @@ protected override void InitializeConfiguration()
                     AgnosticNumberParserFactory.GetParser(AgnosticNumberParserType.Percentage, new DutchNumberParserConfiguration()),
                     new Dutch.PercentageExtractor(options)));
 
-            /*
             RegisterModel<NumberRangeModel>(

```

Comment:
This is needed to run the corresponding tests, right? But we can't merge it in with the recognizer registered.

Reply:
I found this out also yesterday, after my PR... I will try to take some time for this this week, to clean it up and debug the failing tests. There are more tests failing than I thought.

Would prefer not to merge it in + enable tests, without almost full coverage.